tardis/transport/montecarlo/nonhomologous_grid.py

```diff
@@ -7,47 +7,47 @@
 
 
 @njit(**njit_dict_no_parallel)
-def velocity_dvdr(r_packet, numba_model):
+def velocity_dvdr(r_packet, geometry):
     """
     Velocity at radius r and dv/dr of current shell
 
     Parameters
     ----------
     r_packet: RPacket
-    numba_model: NumbaModel
+    geometry: Geometry
 
     Returns
     -----------
     v: float, current velocity
     frac: float, dv/dr for current shell
     """
     shell_id = r_packet.current_shell_id
-    v_inner = numba_model.v_inner[shell_id]
```

Comment:
how did this work? Numba model only had time_explosion?

Reply:
It didn't, this nonhomologous code is quite old now.